tests/console/puppet.pm

```diff
@@ -18,8 +18,12 @@ use utils;
 sub run {
     select_console 'root-console';
     pkcon_quit;
-    my $output      = "puppet cert list --all | grep -woh puppetslave.local > /dev/$serialdev";
-    my $puppet_conf = <<"EOF";
+    if (script_run('zypper se puppet') == 104) {
```

Comment:
I guess we should add version check as well, as we don't want to soft-fail on SLE 12, unless we have SLE 12 specific bug.